### ISSUE TITLE
WCAG 3 is currently missing pause, stop, hide all moving and scrolling content

### ISSUE THREAD

**Issue by julierawe** [w3c/wcag3#734]
### **Issue description**
The current working draft of WCAG 3 is missing essential user controls to manage or dismiss content that moves, scrolls, blinks, auto-updates, or changes without user interaction. This functionality should follow existing WCAG criteria[ 2.2.2 Pause, Stop, Hide](https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html). 

Pause, stop, hide for video/animation appears to be absent in “[Animation and Movement](https://www.w3.org/TR/wcag-3.0/#animation-and-movement)” and “[Images and Media](https://w3c.github.io/wcag3/guidelines/#images-and-media)” of the WCAG 3 draft.

There is a provision that focuses on [pausing/stopping/adjusting the audio in auto-playing audio](https://w3c.github.io/wcag3/guidelines/#page-view-audio-adjustable), but we aren’t seeing anything similar that specifies pausing/stopping/hiding auto-playing video/animation. “User settings supported” has a note that mentions “motion” as one of many examples. Is this the only place where WCAG 3 plans to cover pause, stop, hide for video/animation? 

### **User need missing or not adequately addressed** 
As a user with an attention impairment and impaired memory, I need to avoid distraction. If I lose focus and forget what I am doing, I need reminders of what I was doing, so that I can complete my task. ([MCU 3.5.1](https://www.w3.org/TR/coga-usable/#distractions-user-story), intro)

### **More details**
Users with attention deficits or executive function impairments ([like the Yuki persona](https://www.w3.org/TR/coga-usable/use_cases.html#yuki-a-yoga-teacher-who-has-ad-h-d)) lose focus and cannot concentrate on main content if peripheral content (like carousels or news banners) is moving.

### **Recommendations**

- The user can control all carousels, banners, and animated backgrounds. Possible methods include (1) providing a pause or stop or (2) requiring the user to move the carousel forward or backward (in other words, it doesn't auto-scroll)
- Pausing the movement does not obscure the underlying content or remove the user's ability to manually navigate the carousel/banner.
- Follows [MCU Pattern 4.6.1](https://www.w3.org/TR/coga-usable/#limit-interruptions-pattern) (Limit Interruptions).

### **See also**
Development of the AASPIRE Web Accessibility Guidelines for Autistic Web Users, Dora M. Raymaker, Steven K. Kapp, Katherine E. McDonald, Michael Weiner, Elesia Ashkenazy, and Christina Nicolaidis, Published Online:13 Apr 2019 https://doi.org/10.1089/aut.2018.0020

With thanks,
The COGA Task Force
